docs/explanations/motd_messages.md

```diff
@@ -2,21 +2,47 @@
 
 When the Ubuntu Pro Client (`pro`) is installed on the system, it delivers
 custom messages on ["Message of the Day" (MOTD)](https://wiki.debian.org/motd).
-Those messages are generated directly by two different sources.
+Those messages are generated directly by three different sources.
 
```

Comment:
Once we're done tweaking the doc, could we add a bullet list here with the three sources? (I know the doc isn't long, but it's a helpful thing to have)

Reply:
ack and done